src/pages/repo-new.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import t from 'tcomb-form'
+import _ from 'lodash'
+
+let Form = t.form.Form;
+
+let Repo = t.struct({
+  name: t.Str,
+});
+
+export default React.createClass({
+
+  getInitialState() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).